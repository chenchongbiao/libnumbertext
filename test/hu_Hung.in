123
1823
2021-ben
24-én
í 0
Í 0
NY
?
,
;
„”
kérdőjel?
vessző,
pontosvessző;
„idézőjel”
„𐳐𐳇𐳋𐳯𐳟𐳒𐳉𐳖?”
„𐳐𐳇𐳋𐳯𐳟𐳒𐳉𐳖”?
„𐳐𐳇𐳋𐳯𐳟𐳒𐳉𐳖”,
„𐳐𐳇𐳋𐳯𐳟𐳒𐳉𐳖”;
(parentheses) 0
[parentheses] 0
{parentheses} 0
(parenthesis 0
parenthesis) 0
[parenthesis 0
parenthesis] 0
{parenthesis 0
parenthesis} 0
Händel 0
Molière 0
IQ 0
Apáthy
Apáthyval
Apponyi
Apponyival
Babits
Babitscsal
Báthory
Báthoryval
Batsányi
Batsányival
Batthyány
Batthyányval
Bay 0
Bessenyei
Bessenyeivel
Czuczor
Czuczorral
Csathó
Csathóval
Damjanichcsal
Des​sewffy
Des​sewffyvel
Dsida
Dsidával
Eördögh
Eördöghgel
Eöry
Eöryvel
Eötvös
Eötvössel
Ga​ál
Ga​ál​lal
Háy
Háyval
Joó
Jo​óval
Kass 0
Kazinczy
Kazinczyval
Kéthly
Kéthlyvel
Konko​ly-Thege
Konko​ly-Thegét
Koós
Koóssal
Kossuth
Kossuthtal
Madách
Madáchcsal
Mikszáth
Mikszáthtal
Móricz
Móriczcal
Pais
Paiszsal
Rákóczi
Rákóczival
Széchenyi
Széchenyivel
Széll 0
Tarr 0
Thewrewk
Thewrewkkel
Thököly
Thökölyvel
Ve​ér
Ve​érrel
Verseghy
Verseghyvel
Vörösmarty 0
Vörösmartyt 0
Wass
Wassal 0
Weöres
Weöressel
Wesselényi
Wesselényivel
yard 0
Ybl 0
Zichy
Zichyvel
Zigány
Zigánnyal
