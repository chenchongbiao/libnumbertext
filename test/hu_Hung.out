𐳾𐳼𐳼𐳺𐳺𐳺
𐳿𐳻𐳺𐳺𐳺𐳾𐳼𐳼𐳺𐳺𐳺
𐳺𐳺𐳿𐳼𐳼𐳺-𐳂𐳉𐳙
𐳼𐳼𐳺𐳺𐳺𐳺-𐳋𐳙
𐳑
𐲑
𐲚
⸮
⹁
⁏
⹂‟
𐳓𐳋𐳢𐳇𐳟𐳒𐳉𐳖⸮
𐳮𐳉𐳥𐳥𐳟⹁
𐳠𐳛𐳙𐳦𐳛𐳤𐳮𐳉𐳥𐳥𐳟⁏
⹂𐳐𐳇𐳋𐳯𐳟𐳒𐳉𐳖‟
⹂𐳐𐳇𐳋𐳯𐳟𐳒𐳉𐳖⸮‟
⹂𐳐𐳇𐳋𐳯𐳟𐳒𐳉𐳖‟⸮
⹂𐳐𐳇𐳋𐳯𐳟𐳒𐳉𐳖‟⹁
⹂𐳐𐳇𐳋𐳯𐳟𐳒𐳉𐳖‟⁏
(𐳠𐳀𐳢𐳉𐳙𐳦𐳏𐳉𐳤𐳉𐳤)
[𐳠𐳀𐳢𐳉𐳙𐳦𐳏𐳉𐳤𐳉𐳤]
{𐳠𐳀𐳢𐳉𐳙𐳦𐳏𐳉𐳤𐳉𐳤}
(𐳠𐳀𐳢𐳉𐳙𐳦𐳏𐳉𐳤𐳐𐳤
𐳠𐳀𐳢𐳉𐳙𐳦𐳏𐳉𐳤𐳐𐳤)
[𐳠𐳀𐳢𐳉𐳙𐳦𐳏𐳉𐳤𐳐𐳤
𐳠𐳀𐳢𐳉𐳙𐳦𐳏𐳉𐳤𐳐𐳤]
{𐳠𐳀𐳢𐳉𐳙𐳦𐳏𐳉𐳤𐳐𐳤
𐳠𐳀𐳢𐳉𐳙𐳦𐳏𐳉𐳤𐳐𐳤}
𐲏𐳋𐳙𐳇𐳉𐳖
Molière
IQ
𐲀𐳠𐳁𐳦𐳐
𐲀𐳠𐳁𐳦𐳐𐳮𐳀𐳖
𐲀𐳠𐳛𐳚𐳐
𐲀𐳠𐳛𐳚𐳐𐳮𐳀𐳖
𐲂𐳀𐳂𐳐𐳆
𐲂𐳀𐳂𐳐𐳆𐳆𐳀𐳖
𐲂𐳁𐳦𐳛𐳢𐳐
𐲂𐳁𐳦𐳛𐳢𐳐𐳮𐳀𐳖
𐲂𐳀𐳆𐳁𐳚𐳐
𐲂𐳀𐳆𐳁𐳚𐳐
𐲂𐳀𐳨𐳨𐳁𐳚𐳐
𐲂𐳀𐳨𐳨𐳁𐳚𐳐𐳮𐳀𐳖
𐲂𐳀𐳒𐳐
𐲂𐳉𐳤𐳉𐳚𐳉𐳐
𐲂𐳉𐳤𐳉𐳚𐳉𐳐𐳮𐳉𐳖
𐲄𐳪𐳄𐳛𐳢
𐲄𐳪𐳄𐳛𐳢𐳢𐳀𐳖
𐲆𐳀𐳦𐳜
𐲆𐳀𐳦𐳜𐳮𐳀𐳖
𐲇𐳀𐳘𐳒𐳀𐳙𐳐𐳆𐳆𐳀𐳖
𐲇𐳉𐳰𐳟𐳌𐳐
𐲇𐳉𐳰𐳟𐳌𐳐𐳮𐳉𐳖
𐲇𐳤𐳐𐳇𐳀
𐲇𐳤𐳐𐳇𐳁𐳮𐳀𐳖
𐲞𐳢𐳇𐳞𐳍
𐲞𐳢𐳇𐳞𐳍𐳍𐳉𐳖
𐲟𐳢𐳐
𐲟𐳢𐳐𐳮𐳉𐳖
𐲞𐳦𐳮𐳞𐳤
𐲞𐳦𐳮𐳞𐳤𐳤𐳉𐳖
𐲍𐳁𐳖
𐲍𐳁𐳖𐳖𐳀𐳖
𐲏𐳁𐳒𐳐
𐲏𐳁𐳒𐳐𐳮𐳀𐳖
𐲒𐳜
𐲒𐳜𐳮𐳀𐳖
𐲓𐳀𐳤
𐲓𐳀𐳯𐳐𐳙𐳄𐳐
𐲓𐳀𐳯𐳐𐳙𐳄𐳐𐳮𐳀𐳖
𐲓𐳋𐳦𐳖𐳐
𐲓𐳋𐳦𐳖𐳐𐳮𐳉𐳖
𐲓𐳛𐳙𐳓𐳛𐳖𐳐-𐲦𐳉𐳍𐳉
𐲓𐳛𐳙𐳓𐳛𐳖𐳐-𐲦𐳉𐳍𐳋𐳦
𐲓𐳜𐳤
𐲓𐳜𐳤𐳤𐳀𐳖
𐲓𐳛𐳤𐳪𐳦
𐲓𐳛𐳤𐳪𐳦𐳦𐳀𐳖
𐲘𐳀𐳇𐳁𐳆
𐲘𐳀𐳇𐳁𐳆𐳆𐳀𐳖
𐲘𐳐𐳓𐳥𐳁𐳦
𐲘𐳐𐳓𐳥𐳁𐳦𐳦𐳀𐳖
𐲘𐳜𐳢𐳐𐳄
𐲘𐳜𐳢𐳐𐳄𐳄𐳀𐳖
𐲠𐳀𐳒𐳰
𐲠𐳀𐳒𐳰𐳰𐳀𐳖
𐲢𐳁𐳓𐳜𐳄𐳐
𐲢𐳁𐳓𐳜𐳄𐳐𐳮𐳀𐳖
𐲥𐳋𐳆𐳋𐳚𐳐
𐲥𐳋𐳆𐳋𐳚𐳐𐳮𐳉𐳖
𐲥𐳋𐳖
𐲦𐳀𐳢
𐲦𐳞𐳢𐳞𐳓
𐲦𐳞𐳢𐳞𐳓𐳓𐳉𐳖
𐲦𐳞𐳓𐳞𐳖𐳐
𐲦𐳞𐳓𐳞𐳖𐳐𐳮𐳉𐳖
𐲮𐳋𐳢
𐲮𐳋𐳢𐳢𐳉𐳖
𐲮𐳉𐳢𐳤𐳉𐳍𐳐
𐲮𐳉𐳢𐳤𐳉𐳍𐳐𐳮𐳉𐳖
𐲮𐳞𐳢𐳞𐳤𐳘𐳀𐳢𐳦𐳐
𐲮𐳞𐳢𐳞𐳤𐳘𐳀𐳢𐳦𐳐𐳦
𐲮𐳀𐳤
𐲮𐳀𐳤𐳤𐳀𐳖
𐲮𐳞𐳢𐳞𐳤
𐲮𐳞𐳢𐳞𐳤𐳤𐳉𐳖
𐲮𐳉𐳤𐳉𐳖𐳋𐳚𐳐
𐲮𐳉𐳤𐳉𐳖𐳋𐳚𐳐𐳮𐳉𐳖
𐳒𐳀𐳢𐳇
𐲑𐳂𐳖
𐲯𐳐𐳆𐳐
𐲯𐳐𐳆𐳐𐳮𐳉𐳖
𐲄𐳐𐳍𐳁𐳚
𐲄𐳐𐳍𐳁𐳚𐳚𐳀𐳖
